content/blog/hacking-build-process.md

```diff
@@ -0,0 +1,344 @@
+    mp.messaging.outgoing.notify-event.topic=notify-event
+
+    quarkus.kafka.devservices.enabled=false
+
+    # TLS support
+    kafka.bootstrap.servers=kafka-cluster-3-kafka-bootstrap.orchestrator.svc:9093
+    mp.messaging.connector.smallrye-kafka.security.protocol=SSL
+    # Specify the enabled TLS protocols (forcing TLSv1.2)
+    mp.messaging.connector.smallrye-kafka.ssl.enabled.protocols=TLSv1.2
+
+    # Truststore configuration (so the client can validate the broker's certificate)
+    mp.messaging.connector.smallrye-kafka.ssl.truststore.location=/deployment/certs/truststore.jks
+    mp.messaging.connector.smallrye-kafka.ssl.truststore.password=password
+```
+
+Make sure to update the Kafka configuration, referenced secret path, and password to match your environment.
+
+Here's the complete SonataFlow CR with our additions:
+
+```yaml
+  podTemplate:
+    container:
+      image: quay.io/orchestrator/demo-poc-kafka-logic:latest
+      volumeMounts:
+        - name: truststore-volume
+          mountPath: /deployment/certs
+          readOnly: true
+      resources: {}
+    volumes:
+      - name: truststore-volume
+        secret:
+          secretName: kafka-truststore      
+  persistence:
+    postgresql:
+      secretRef:
+        name: sonataflow-psql-postgresql
+        userKey: postgres-username
+        passwordKey: postgres-password
+      serviceRef:
+        name: sonataflow-psql-postgresql
+        port: 5432
+        databaseName: sonataflow
+        databaseSchema: lock-flow
+```
+
+Now we can deploy the manifests to the cluster. The files are numbered for a reason - the configmap must be applied first, otherwise, the SonataFlow operator will generate an empty configmap for the workflow:
+
+```bash
+oc create -f manifests/01-configmap_lock-flow-props.yaml -n sonataflow-infra
+oc create -f manifests/02-sonataflow_lock-flow.yaml -n sonataflow-infra
+```
+
+Let's watch for the workflow pod status to verify it's running:
+
+```bash
+oc get pod -n sonataflow-infra -l app=lock-flow
+```
+
+You should see output like this:
+
+```
+NAME                        READY   STATUS    RESTARTS   AGE
+lock-flow-5cbd5dbdb-2g7wz   1/1     Running   0          13m
+```
+
+## Testing the Workflow
+
+Now we can test the workflow by emitting cloud events to trigger it. Note that in this example, we're not using the Orchestrator to invoke workflows, since it can only invoke workflows via HTTP endpoints. However, we can monitor the workflow's progress from the workflow runs tab.
+
+### Setting Up the Kafka Producer with TLS
+
+To invoke the workflow, we'll use Kafka's producer script. Since we're using TLS, we need to have the `truststore.jks` file available.
+
+> In this example, we're using Kafka installed on an OpenShift cluster with the Strimzi operator.
+
+First, copy or mount the `truststore.jks` file to the Kafka cluster broker pod. Then create a properties file at `/tmp/client-ssl.properties`:
```

Comment:
Same, would add a new line before `Then`:
```
First, copy or mount the `truststore.jks` file to the Kafka cluster broker pod. 

Then create a properties file at `/tmp/client-ssl.properties`: ...
```

Reply:
And maybe provide the command that create the file: as we are running within the Kafka pod, `vi` and other text editor are not available, providing the `echo` command will be a nice attention to the reader, in case they need it.